plugins/Wallet/index.html

```diff
@@ -12,6 +12,6 @@
 	<body>
 		<!-- React root -->
 		<div id="react-root"></div>
-		<script src="js/index.js"></script>
+		<script src="../../dist/plugins/Wallet.js"></script>
```

Comment:
could we modify it such that the compiled js was placed inside the plugin directory, instead of a globally shared directory?